packages/editor-ui/src/modules/nodeTypes.ts

```diff
@@ -26,25 +26,47 @@ const module: Module<INodeTypesState, IRootState> = {
 	},
 	getters: {
 		allNodeTypes: (state): INodeTypeDescription[] => {
-			return Object.values(state.nodeTypes);
+			return Object.values(state.nodeTypes).reduce<INodeTypeDescription[]>((allNodeTypes, nodeType) => {
+				const nodeVersions = Object.keys(nodeType).map(Number);
+
+				if (nodeVersions.length === 0) {
+					return [...allNodeTypes, nodeType[DEFAULT_NODETYPE_VERSION]];
+				}
```

Comment:
this logic does not make sense to me... I think you will always have versions since you are ensuring that when you are setting them.. also if `nodeVersions.length === 0` then `nodeType[1]` will be null